PS C:\Users\User> & C:/Python/Python310/python.exe "c:/Users/User/Desktop/Основы сетей/server.py"
Server if listening...
Connected with ('192.168.0.6', 54982)
Nickname is Sts
Connected with ('192.168.0.6', 54991)
Nickname is Nick
Connected with ('192.168.0.6', 55001)
Nickname is Vova
Connected with ('192.168.0.6', 55012)
Nickname is Bill
Connected with ('192.168.0.6', 55017)
Nickname is Nik

 *  History restored 

PS C:\Users\User> 
 *  History restored 

PS C:\Users\User> git clone https://github.com/zsv-job/remote-test-1.git
Cloning into 'remote-test-1'...
remote: Enumerating objects: 3, done.
remote: Counting objects: 100% (3/3), done.
remote: Total 3 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
Receiving objects: 100% (3/3), done.
PS C:\Users\User> cd 'C:\Users\User\Desktop\Git2\'     
PS C:\Users\User\Desktop\Git2> dir


    Каталог: C:\Users\User\Desktop\Git2


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        15.09.2024     15:32                remote-test-1
d-----        18.09.2024     11:53                remote-test-1-1


PS C:\Users\User\Desktop\Git2> git clone https://github.com/zsv-job/remote-test-1.git
fatal: destination path 'remote-test-1' already exists and is not an empty directory.
PS C:\Users\User\Desktop\Git2> dir


    Каталог: C:\Users\User\Desktop\Git2


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----        15.09.2024     15:32                remote-test-1
d-----        18.09.2024     11:53                remote-test-1-1


PS C:\Users\User\Desktop\Git2> git clone https://github.com/zsv-job/remote-test-1.git
Cloning into 'remote-test-1'...
remote: Enumerating objects: 3, done.
remote: Counting objects: 100% (3/3), done.
remote: Total 3 (delta 0), reused 0 (delta 0), pack-reused 0 (from 0)
Receiving objects: 100% (3/3), done.
PS C:\Users\User\Desktop\Git2> git remote -v
PS C:\Users\User\Desktop\Git2> git remote -v
PS C:\Users\User\Desktop\Git2> git clone https://github.com/zsv-job/remote-test-1.git
fatal: destination path 'remote-test-1' already exists and is not an empty directory.
PS C:\Users\User\Desktop\Git2> git remote -v
PS C:\Users\User\Desktop\Git2> git remote -v
PS C:\Users\User\Desktop\Git2> git remote add origin https://github.com/zsv-job/remote-test-2.git
PS C:\Users\User\Desktop\Git2> git remote -v
origin  https://github.com/zsv-job/remote-test-2.git (fetch)
origin  https://github.com/zsv-job/remote-test-2.git (push)
PS C:\Users\User\Desktop\Git2> git push -u origin master
Enumerating objects: 26, done.
Counting objects: 100% (26/26), done.
Delta compression using up to 16 threads
Compressing objects: 100% (18/18), done.
Writing objects: 100% (26/26), 4.83 KiB | 1.61 MiB/s, done.
Total 26 (delta 5), reused 0 (delta 0), pack-reused 0 (from 0)
remote: Resolving deltas: 100% (5/5), done.
To https://github.com/zsv-job/remote-test-2.git
 * [new branch]      master -> master
branch 'master' set up to track 'origin/master'.
PS C:\Users\User\Desktop\Git2>


